### DISCUSSION TITLE
Add how-to-install (or something similar) to `README.md`

### DISCUSSION THREAD

**Discussion by NimaSarajpoor** [blue-yonder/tsfresh#948] [General]
I think how-to-install is missing! Is that intentional?

**Comment by nils-braun** [comment]
We have an instruction on our documentation page (https://tsfresh.readthedocs.io/en/latest/text/quick_start.html) which is linked from the README and in general it is a rather "standard" pip install.
Do you think the README would benefit from this information? If yes, feel free to do a PR for this! 
Thanks :)

**Reply by NimaSarajpoor** [reply]
I went and checked the readme again. Now that you mentioned it, I noticed the link you were talking about. So, I might have spoken too soon! I assume most users probably go through the intro and they will eventually see that link 🙂
